src/drive/lib/encryption.js

```diff
@@ -1,4 +1,5 @@
 import get from 'lodash/get'
+import flag from 'cozy-flags'
```

Comment:
- please add the flag  in the list of the development flag: https://github.com/cozy/cozy-drive/blob/master/src/lib/flags.js
- enable the feature in prod' for cozy internal here: https://bender.cozycloud.cc/contexts/prod/cozy_internal

Reply:
* Ah right, I forgot the existence of this list
* We don't want to enable this feature yet

Reply:
even for Cozy internal ? (it means the employee of cozy)